packages/cyberstorm/src/components/PackageList/PackageList.tsx

```diff
@@ -22,13 +23,6 @@ interface Props {
 
 const PER_PAGE = 20;
 
-/**
- * Fetches packages based on props and shows them as a list.
```

Comment:
Nitpick: I'm not saying this comment was very helpful, but I'd argue it was more helpful than the generic copy pasted "Cyberstorm X component" comments we're allowing to exists :D

Reply:
Didn't mean to remove this 🤦 I'll add it back